src/descriptor/mod.rs

```diff
@@ -783,5 +801,14 @@ mod test {
             result.unwrap_err(),
             DescriptorError::HardenedDerivationXpub
         ));
+
+        let descriptor = "wsh(multi(2,tpubD6NzVbkrYhZ4XHndKkuB8FifXm8r5FQHwrN6oZuWCz13qb93rtgKvD4PQsqC4HP4yhV3tA2fqr2RbY5mNXfM7RxXUoeABoDtsFUq2zJq6YK/0/*,tpubD6NzVbkrYhZ4XHndKkuB8FifXm8r5FQHwrN6oZuWCz13qb93rtgKvD4PQsqC4HP4yhV3tA2fqr2RbY5mNXfM7RxXUoeABoDtsFUq2zJq6YK/1/*))";
+        let result = into_wallet_descriptor_checked(descriptor, &secp, Network::Testnet);
+
+        assert!(result.is_err());
```

Comment:
I don't think we need this, the `unwrap_err` panics if its not an error.

Reply:
I think on line 810 he's unwrapping it to assert it's the expected `DescriptorError::DuplicatedKeys`

Reply:
Yes, there's also the previous assertion to ensure that it's actually an error. If any of those assertions or unwraps cause a panic the test will fail, which is what we want if something weird is going on.